### DISCUSSION TITLE
RoundCube Config Menu Link?

### DISCUSSION THREAD

**Discussion by Xtrato01** [YetiForceCompany/YetiForceCRM#17468] [How to]
Hi Everyone , i must be missing something , with the new version  7.0.> , i can not seem to find the roudcube link to configure it , i have downloaded the library and enabled MyMail , but , on previous versions it was under the Mail tools.

Thanks!

**Comment by rskrzypczak** [comment]
In fact, there is no configuration for roundcube anymore. You configure mail servers and then choose which server to use in your mail accounts.
You log in to the roundcube email client automatically for configured email accounts.

**Comment by Xtrato01** [comment]
alright that is good to know , great thank you !

So ive set up my mail server , and its not working , it keeps saying Log in Failure. ive set the other SMTP Configuration and that does work well ,it logs in a sends a test email.

Anything i may be missing?

**Reply by mariuszkrzaczkowski** [reply]
attach more information, logs

**Comment by Xtrato01** [comment]
i was able to actually log in - to my personal email with the "MAILBOX" Tab - however , i remember i was able to add multiple emails and access trough drop down - i dont seem to able to do that anymore

also , im trying to add another user with an email but im getting the following error:

![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/90271495/3fd675ab-9d09-4cf6-bae4-a5142bf8c167)

**Reply by rskrzypczak** [reply]
You are able. In version 7.0, email accounts were simply divided into private and "public" ones. Only you can log in to private ones. You can set who else can log in to "public mailbox" in the administration panel `Mail tools/Mail – Auto Login`.
You add public accounts in the panel `Mail tools/Mail – Scanner`

**Comment by Xtrato01** [comment]
ok great, i was able to do that - thanks for the tip - a better documentation** is needed lol , where can i donate?

Another question , how can i enable an "Account" to download automatically "download' emails to be shown in its profile?

Regards

**Reply by rskrzypczak** [reply]
Simply set value to `Folders`, `Scanner actions`, `Search fields` field in mail account, to active scan emails
